dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/user/UserLookupController.java

```diff
@@ -85,6 +85,7 @@ public UserLookups lookUpUsers( @RequestParam String query )
     {
         UserQueryParams params = new UserQueryParams()
             .setQuery( query )
+            .setCanSeeOwnUserAuthorityGroups( true )
```

Comment:
I see that the current method name is "canSeeOwnUserAuthorityGroups" - but the authority is CAN_GRANT_OWN_USER_AUTHORITY_GROUPS. I realize that this name was inherited from the previous behavior, but it seems that we could do a little cleanup here. All users should be able to "**see**" their own groups and autos - but they should not be able to "**grant**" it.
This is not a big deal, and I'm not sure how much work it would be to rewrite this, I'll let you decide wether to implement a change to the naming and logic, or wether to keep it like this.

Reply:
I would rather have it included before hard freeze tonight than have it perfect :)

Reply:
@larshelge I think you added the logic in question. Maybe you can help us understand why it is needed or if this was a misunderstanding and always just should have targeted just the granting.

But yes, this is a different change we can and should make in another PR - also to get this in 2.37?

Reply:
I think this is in fact correct, as this method is just fetching/reading user information. We have other endpoints for writing/modifying/granting access.